.circleci/config.yml

```diff
@@ -659,7 +659,7 @@ jobs:
                 path: ~/project
             - attach_workspace:
                 at: /tmp/workspace
-            - run: apt update -y && apt install -y nodejs npm curl
+            - run: apt update -y && apt install -y nodejs npm curl sudo
```

Comment:
Possibly we should add `sudo` as a dependency to our debian package then?

Reply:
https://github.com/digitalfabrik/entitlementcard/blob/release-mai-2024/.circleci/src/jobs/pack_backend.yml#L11C232-L11C255

Reply:
is that a feature circle-ci provides? i couldn't find any documentation how to do that in circleci @michael-markl 
Or are there specific debian image including sudo, which i also couldn't find in docker image documentation

Reply:
Not really happy with this solution. Why do we want to add sudo, as everything runs on debian we can just remove the sudo's from the script. Also only adding sudo as a package seems not to be sufficent, as we need to add the current user to sudoers group and log in and out.

Reply:
currently when i try to switch user on staging server i need sudo. thats why the command is like that for migration in the post install script

Reply:
> is that a feature circle-ci provides? i couldn't find any documentation how to do that in circleci @michael-markl 

The line I referred to uses the pack deb script which accepts dependencies for the deb package. Currently the only dependency is the Java runtime. We could add sudo there.

(Sorry for slow response, I'm on vacation)